spec/features/session_behaviour_spec.rb

```diff
@@ -69,6 +69,26 @@
     end
   end
 
+  context 'when an active session already exists' do
+    let(:user) { create(:local_authority_user) }
+    let(:validate_token_url) { validate_token_url_for(user) }
+
+    before do
+      FeatureFlag.deactivate(:extra_mobile_data_offer)
+      visit validate_token_url
+    end
+
+    scenario 're-using the same magic-link redirects to the home page for user' do
+      visit validate_token_url
+      expect(page).to have_current_path(download_responsible_body_bt_wifi_vouchers_path)
+    end
+
+    scenario 'using a new magic link redirects to the home page for user' do
+      sign_in_as(user)
+      expect(page).to have_current_path(download_responsible_body_bt_wifi_vouchers_path)
+    end
+  end
+
```

Comment:
I'd suggest we code the test against the `extra_mobile_data_offer` being enabled, since we're likely to remove it (and leave the functionality permanently on) within the week.

Reply:
OK thanks, that makes sense. I deactivated that feature as it was the first way I found that I could get a path I could test against.

Reply:
^ what @benilovj said, otherwise fine 👍